host/traceh.c

```diff
@@ -331,6 +348,18 @@ void oe_log_message(bool is_enclave, oe_log_level_t level, const char* message)
     {
         initialize_log_config();
     }
+
+    if (_oe_log_callback)
```

Comment:
This check seems to be redundant. Or do you mean `_oe_log_context` (can `_oe_log_context` be `NULL`)?

Reply:
_oe_log_context can be null. Only while _oe_log_callback is set the messages are redirected, or messages will be processed in oe_log_message as before(no redirect)